src/utils.py

```diff
@@ -12,30 +12,24 @@ def create_directory(path):
 
 # Tiled Saving
 def save_seg_to_tiled(seg_result,
-                      data_tiled_uri,
-                      mask_tiled_uri,
+                      tiled_dataset,
                       seg_tiled_uri,
                       seg_tiled_api_key,
-                      container_keys,
                       uid,
                       model,
                       ):
     
     last_container = from_uri(seg_tiled_uri, api_key=seg_tiled_api_key)
-    container_keys.append(uid)
-    
-    for key in container_keys:
-        if key not in last_container.keys():
-            last_container = last_container.create_container(key=key)
-        else:
-            last_container = last_container[key]
+    last_container = last_container.create_container(key=uid)
 
     metadata={
-        'data_tiled_uri': data_tiled_uri,
-        'mask_uri': mask_tiled_uri, 
+        'data_tiled_uri': tiled_dataset.data_tiled_uri,
+        'mask_uri': tiled_dataset.mask_tiled_uri, 
+        'mask_idx': tiled_dataset.mask_idx,
```

Comment:
As @Wiebke requested for the Inference-Preview mode, we would like to also save the `mask_idx` as part of the result metadata.

Reply:
That is a good point, Tanny.
I thought it would be a good idea to redundantly include these here to be able to directly map segmentation result to original data, but this can indeed also be retrieved through the mask uri. 

On a different note, if `mask_tiled_uri` is not given (in the segmental all case), does Tiled give us errors when one of the meta data values is `None`?